build/release-tars/BUILD.bazel

```diff
@@ -149,14 +149,14 @@ pkg_tar(
 ) for arch in archs] for os, archs in UBI_PLATFORMS.items()]
 
 [[pkg_tar(
-    name = "cert-manager-ctl-%s-%s" % (os, arch),
+    name = "kubectl-cert-manager-%s-%s" % (os, arch),
     srcs = [
         ":license-targets",
         "//:version",
     ],
     extension = "tar.gz",
     mode = "0644",
-    package_dir = "cert-manager-ctl-%s-%s" % (os, arch),
+    package_dir = "kubectl-cert-manager-%s-%s" % (os, arch),
     tags = [
         "manual",
         "no-cache",
```

Comment:
(I can't comment far down enough on the line I need to, but..)

https://github.com/jetstack/cert-manager/blob/13e7b7d2686c1d0bff030f7d2b5f6c30a5ad711c/build/release-tars/BUILD.bazel#L103

The `_ctl` target has a `package_dir` of `client/ctl` which is not what we want. This `package_dir` setting on `:_ctl` should be removed altogether which should fix up your weird layout issues